src/main/java/FruitMain.java

```diff
@@ -0,0 +1,46 @@
+import dao.WokWithStorageImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.convert.ConvertData;
+import service.convert.ConvertDataImpl;
+import service.filework.AddDataToFileImpl;
+import service.filework.GetDataFromFile;
+import service.filework.GetDataFromFileImpl;
+import strategy.HandlersStore;
+import strategy.StrategyImplementation;
+import strategy.StrategyImplementationImpl;
+import strategy.handler.BalancesHandlerImpl;
+import strategy.handler.Handler;
+import strategy.handler.PurchaseHandlerImpl;
+import strategy.handler.ReturnHandlerImpl;
+import strategy.handler.SupplyHandlerImpl;
+
+public class FruitMain {
+    private static final String PATH_TO_DATA = "src/main/resources/fileFuringDay.csv";
+    private static final String PATH_TO_REPORT = "src/main/resources/fileReport.csv";
+
+    public static void main(String[] args) {
+        GetDataFromFile getDataService = new GetDataFromFileImpl();
+        List<String> data = getDataService.getFromStorage(PATH_TO_DATA);
+        ConvertData convertData = new ConvertDataImpl();
+        List<FruitTransaction> dataConvert = convertData.convert(data);
+        StrategyImplementation implementStrategy =
+                new StrategyImplementationImpl(new HandlersStore(addPatternHandler()));
+        implementStrategy.strategy(dataConvert);
+        WokWithStorageImpl storageService = new WokWithStorageImpl();
```

Comment:
WokWithStorageImpl  - do refactoring the name. And it`s will be better to create instance with reference to the parent class type.
```suggestion
        WorkWithStorageDB storageService = new WorkWithStorageImpl();
```